lib/src/main/java/com/tbruyelle/rxpermissions/Trigger.java

```diff
@@ -0,0 +1,9 @@
+package com.tbruyelle.rxpermissions;
+
+final class Trigger {
+    static final Trigger INSTANCE = new Trigger();
```

Comment:
Ok, but do we really need an extra class for that ? Why not a simple field in the `RxPermissions` class : 

```java
static final Object trigger = new Object();
```